npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/registry_finder.rb

```diff
@@ -232,6 +238,7 @@ def global_registry
           "https://registry.npmjs.org"
         end
         # rubocop:enable Metrics/PerceivedComplexity
+        # rubocop:enable Metrics/AbcSize
```

Comment:
If the above comments are correct, this method does not need to change, so you need to undo the RuboCop skips as well.